readme.md

```diff
@@ -174,6 +174,31 @@ const memGot = mem(got, {cache});
 })();
 ```
 
+### WeakMap cache with multiple arguments
```

Comment:
I feel like this shouldn't be here. The `cacheKey` section already mentions:

> You can have it cache **all** the arguments by value with JSON.stringify, if they are compatible:

So you should extend that, which a minimal example like the 2 existing ones, clarifying _when_ you'd want to use this over a regular stringifier.

Reply:
I tried to do this when addressing this comment https://github.com/sindresorhus/mem/pull/51#discussion_r362447271, is it better now?

Reply:
No, it needs to be in the `cacheKey` section **and** the example needs to be trimmed to match the other two examples in that description. We don't need a full example with unrelated requires, it makes it harder to follow.

Reply:
Are you sure about this? This use case covers both `cache` and `cacheKey` options.

Reply:
Indeed. Then all examples should be moved outside `cacheKey`. They're all about different ways to use the cache, so they should be listed together and clearly.

I think the section should be called: **Using the right cache for your function** or something along those lines. In it, probably you can start with that's in `cacheKey` now and then continue with another minimal example using DeepWeakMap